tiled/catalog/adapter.py

```diff
@@ -1408,7 +1420,7 @@ def structure_family(query, tree):
 CatalogNodeAdapter.register_query(StructureFamilyQuery, structure_family)
 CatalogNodeAdapter.register_query(SpecsQuery, specs)
 CatalogNodeAdapter.register_query(FullText, full_text)
-# TODO: Regex
```

Comment:
I am dropping this TODO because SQLite does not support regex (out of the box) and `LIKE` covers the most common use case.

Of course it could be revisited, but I don't think this code comment is helping anything now.